save-backend/src/main/kotlin/org/cqfn/save/backend/repository/ExecutionRepository.kt

```diff
@@ -1,10 +1,17 @@
 package org.cqfn.save.backend.repository
 
 import org.cqfn.save.entities.Execution
+import org.cqfn.save.entities.Project
 import org.springframework.stereotype.Repository
 
 /**
  * Repository of execution
  */
 @Repository
-interface ExecutionRepository : BaseEntityRepository<Execution>
+interface ExecutionRepository : BaseEntityRepository<Execution> {
+    /**
+     * @param project
+     * @return list of executions
+     */
+    fun getAllByProject(project: Project): List<Execution>
```

Comment:
```suggestion
    fun getAllByProjectNameAndProjectOwner(name: String, owner: String): List<Execution>
```
I think spring-data allows to write what we need in one call